File: workspace-type-definer-and-assigner/type_definer_and_assigner/src/main/java/brss/type_definer_and_assigner/A_Temperature_Unit.java
package brss.type_definer_and_assigner;

public class A_Temperature_Unit {

	private A_Temperature_Scale the_temperature_scale;
	
	public A_Temperature_Unit(A_Temperature_Scale the_temperature_scale_to_use) {
		
		this.the_temperature_scale = the_temperature_scale_to_use;
		
	}
	
	public A_Temperature_Scale The_Temperature_Scale() {
		
		return this.the_temperature_scale;
		
	}
	
}

File: type_definer_and_assigner/type_definer_and_assigner/A_Temperature_Unit.cpp
#include "A_Temperature_Scale.h"
#include "A_Temperature_Unit.h"


namespace type_definer_and_assigner {

	A_Temperature_Unit::A_Temperature_Unit() {

	};

	A_Temperature_Unit::A_Temperature_Unit(A_Temperature_Scale the_temperature_scale_to_use) {
		this->the_temperature_scale = the_temperature_scale_to_use;
	};

	A_Temperature_Scale A_Temperature_Unit::The_Temperature_Scale() {
		return this->the_temperature_scale;
	};

};